iterative/aws/provider.go

```diff
@@ -32,6 +31,15 @@ func ResourceMachineCreate(ctx context.Context, d *schema.ResourceData, m interf
 	securityGroup := d.Get("aws_security_group").(string)
 	spot := d.Get("spot").(bool)
 	spotPrice := d.Get("spot_price").(float64)
+
+	metadata := map[string]string{
+		"Name": d.Get("name").(string),
+		"Id":   d.Id(),
```

Comment:
Should we really provide any defaults?

Reply:
The  [`Name` tag](https://docs.aws.amazon.com/whitepapers/latest/tagging-best-practices/standardize-names-for-aws-resources.html) is being used by AWS to name instances in the console, but the `Id` tag doesn't name any sense and should probably be removed unless users ask for it.

Reply:
Let me review it but its the way to relate the runner and the machine, as far as I remember

Reply:
Id and Name are different, not saying that we should not unify them or check. As far as I remember we also had an open issue to remove name in the runner and just stay with tags

Reply:
https://github.com/iterative/cml/issues/525

Reply:
Then, let's leave this as–is, fully backwards compatible, and then address https://github.com/iterative/cml/issues/525 separately.